bombastic/api/src/server.rs

```diff
@@ -408,3 +409,61 @@ async fn delete_sboms(
 
     Ok(HttpResponse::NoContent().finish())
 }
+
+/// Search for a status of sbom using a free form search query.
+///
+/// See the [documentation](https://docs.trustification.dev/trustification/user/retrieve.html) for a description of the query language.
+#[utoipa::path(
+    get,
+    tag = "bombastic",
+    path = "/api/v1/sbom/status",
+    responses(
+    (status = 200, description = "Search completed"),
+    (status = BAD_REQUEST, description = "Bad query"),
+    (status = 401, description = "Not authenticated"),
+    ),
+    params(
+    ("q" = String, Query, description = "Search query"),
+    )
+)]
+#[get("/sbom/status")]
+async fn sbom_status(
```

Comment:
I have a suggestion: there is no need to create a new endpoint, we can just utilize the existing one.[1] 

[1] https://github.com/trustification/trustification/blob/621fc8abc1b67a25686e635a1f9ac5319a845b82/bombastic/api/src/server.rs#L235